app/src/main/java/com/infomaniak/drive/ui/fileList/preview/PreviewPictureFragment.kt

```diff
@@ -85,18 +87,13 @@ class PreviewPictureFragment : PreviewFragment() {
     }
 
     private fun loadImage() = with(binding) {
-
-        suspend fun setImageDrawable(drawable: Drawable) = withContext(Dispatchers.Main) {
-            imageView.setImageDrawable(drawable)
-        }
-
         val imageLoader = Coil.imageLoader(requireContext())
         val thumbnailPreviewRequest = buildThumbnailPreviewRequest()
         val previewRequest = buildPreviewRequest()
 
         viewLifecycleOwner.lifecycleScope.launch(Dispatchers.IO) {
-            imageLoader.execute(thumbnailPreviewRequest).drawable?.let { setImageDrawable(it) }
-            imageLoader.execute(previewRequest).drawable?.let { setImageDrawable(it) }
+            imageLoader.execute(thumbnailPreviewRequest)
+            imageLoader.execute(previewRequest)
```

Comment:
Looks like we can use coil's `load()` method on the image view directly so we don't have to set the Drawable nor start the Animatable manually